src/evaluators.js

```diff
@@ -155,6 +140,27 @@ export function createSafeEvaluatorFactory(unsafeRec, safeGlobal, transforms) {
     // of Function in any compartment of the same root realm.
     setPrototypeOf(safeEval, unsafeFunction.prototype);
 
+    // todo (shim limitation): scan endowments, throw error if endowment
+    // overlaps with the const optimization (which would otherwise
+    // incorrectly shadow endowments), or if endowments includes 'eval'. Also
+    // prohibit accessor properties (to be able to consistently explain
```

Comment:
This comment is stale. Nothing in this PR causes it to be stale, since it is in a todo anyway. However, our safe module plans depend on our ability to install accessors on the `scopeTarget`.

Reply:
I'll update the comment.